app/CMakeLists.txt

```diff
@@ -93,6 +93,8 @@ target_compile_definitions(opus PRIVATE OPUS_WILL_BE_SLOW=1) # libopus will warn
 
 # Cubeb
 set(USE_OPENSL ON)
+set(LAZY_LOAD_LIBS OFF)
```

Comment:
just saying but isn't it 
set(USE_LAZY_LOAD_LIBS OFF)

saying it cuz i had a cmake error when compiling with that line